class.jetpack-cli.php

```diff
@@ -304,8 +305,43 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
 				break;
+			case 'sync-checksum':
+				global $wpdb;
+				$option = 'jetpack_callables_sync_checksum';
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites  = get_sites( array( 'number' => 1000 ) );
+					$count_fixes  = 0;
+					foreach ( $sites as $site ) {
+						switch_to_blog( $site->blog_id );
+						$count = (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->options WHERE option_name = %s", $option ) );
```

Comment:
should we LIMIT the query and re run it if there are things left?

Reply:
Some sort of retry? 
Do you think that is likely to be an issue?

Reply:
Also a suggestion: we may be better off using `MAX(id)` for queries like this if we don't plan to actually use the count, but only need it for checksum purposes.

Reply:
@zinigor We are trying to see how many option so that type exists. So I am not sure if MAX(id) would help with this.

Reply:
hmm, I thought you were deleting here, disregard my comment